compensating-transactions/bank/src/main/java/org/jboss/narayana/quickstarts/compensations/bank/Data.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.narayana.quickstarts.compensations.bank;
+
+import org.jboss.narayana.compensations.api.CompensationScoped;
+
+import java.io.Serializable;
+
+@CompensationScoped
+public class Data implements Serializable {

```

Comment:
Wha about adding to  javadoc here an info that the scoped bean serves as state holder bound to the active compensating transaction?